src/Network.php

```diff
@@ -1,6 +1,8 @@
 <?php
 namespace IPTools;
 
+use ReturnTypeWillChange;
```

Comment:
What about this line? Does it cause a conflict with php 5?

Reply:
No https://3v4l.org/BNoFr#v5.6.40

BTW we should drop php5 support as it's EOLed for a long time already